std/heap.zig

```diff
@@ -196,10 +196,10 @@ pub const ArenaAllocator = struct {
     pub allocator: Allocator,
 
     child_allocator: *Allocator,
-    buffer_list: std.LinkedList([]u8),
+    buffer_list: std.linked_list.SinglyLinkedList([]u8),
```

Comment:
You can make it `std.SinglyLinkedList`

Reply:
Why wouldn't we want to namespace all the various linked list alternatives under `std.linked_list`?